### PR TITLE
Share/share button action

### PR DESCRIPTION
**Pull request by aihazm**

### CODE REVIEW

File: public/highscore/highscore.js [hunk 1]
```diff
@@ -3,20 +3,43 @@
  */
 var out;
 function init () {
-	for ( var n = 0; n < data.length; n++) {
-		var placing = n + 1;
```
Comment (ghost) [inline public/highscore/highscore.js:7]: Declare 'placing' outside of loop. Performance.

File: public/spotify-session-app.js [hunk 2]
```diff
@@ -139,13 +160,32 @@ $.post( 'main', {auth: true, code: getUrlVars().code}, function ( data ) {
...
-					updateKpi( reply.model.layout.qHyperCube.qDataPages[0].qMatrix[0][0].qNum );
-					$(".spin").remove();
+
+				function getAvgPopularity () {
+					sessionApp.createGenericObject( {
+						avg: {
+							qStringExpression: "=Avg(popularity)"
+						}
+					}, function ( reply ) {
+						updateKpi( reply.avg );
```
Comment (ghost) [inline public/spotify-session-app.js:168]: Null checking needed here?

**Review by ghost — APPROVED** [review]
Minor issues, can be ignored in this development run.

File: public/highscore/highscore.js (hunk 1 above)
Comment (aihazm) [reply public/highscore/highscore.js:7]: this code is removed

**Review by annloouise — APPROVED** [review]

**Pull request merged**